### PR TITLE
TSS 424 random content updates

### PR DESCRIPTION
**Pull request by jason-trade**

### CODE REVIEW

**Review by ferozerub — APPROVED** [review]
Needs a complete smoke test on UAT. Minor changes though lots of them

File: templates/barriers/edit/commodities.html [hunk 1]
```diff
@@ -109,7 +109,12 @@ <h3>HS commodity codes to add to this barrier</h3>
                 <input type="hidden" name="countries" value="{{ confirmed_commodity.country.id }}" />
                 <input type="hidden" name="trading_blocs" value="{{ confirmed_commodity.trading_bloc.code }}" />
             {% endfor %}
+
+            <button name="action" value="save" class="govuk-button" data-module="govuk-button">Save and return</button>
+            <button class="form-cancel govuk-button button-as-link" name="action" value="cancel">Cancel</button>
+
```
Comment (Santino-Trade) [inline templates/barriers/edit/commodities.html:115]: New buttons? We'll have to pay special attention to this bit - make sure the backend plays ball.

File: templates/barriers/edit/tags.html [hunk 2]
```diff
@@ -34,7 +34,27 @@ <h3>{{ form.tags.label }}</h3>
             </fieldset>
         </div>
 
-        <input type="submit" value="Save changes" class="govuk-button">
+        {% if current_user|has_permission:"set_topprioritybarrier" %}
```
Comment (Santino-Trade) [inline templates/barriers/edit/tags.html:37]: This bit seems quite chunky - a bit more than content updates - was this your change, @jason-trade ?

Comment (Santino-Trade) [reply templates/barriers/edit/tags.html:37]: Ah - i remember - we removed top-priority tagging from the tagging page, this shouldn't be here

File: templates/barriers/edit/commodities.html (hunk 1 above)
Comment (jason-trade) [reply templates/barriers/edit/commodities.html:115]: Shouldn't be new buttons, just new button text. There was a conflict with Master - maybe I didn't resolve it correctly?

Comment (jason-trade) [reply templates/barriers/edit/commodities.html:115]: ie. from 'Save and exit to barrier' to 'Save and return'

File: templates/barriers/edit/tags.html (hunk 2 above)
Comment (jason-trade) [reply templates/barriers/edit/tags.html:37]: Oh weird, my change the value from 'Save changes' to 'Save and return' only
<input type="submit" value="Save and return" class="govuk-button">

**Comment by Santino-Trade** [comment]
Merged

**Pull request closed without merging**